<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QCheckBox" name="onTopCheckBox">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>界面置顶</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="selectionCheckBox">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>启用划词</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="statusEN">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="statusZH">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyZHBtn">
           <property name="font">
            <font>
             <family>黑体</family>
            </font>
           </property>
           <property name="text">
            <string>复制中文</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modeBox">
           <property name="font">
            <font>
             <family>黑体</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>英译汉</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>汉译英</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Both</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="engineBox">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Google</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DeepL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OpenAI</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="screenshotBtn">
           <property name="font">
            <font>
             <family>黑体</family>
            </font>
           </property>
           <property name="text">
            <string>截图翻译</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitBtn">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>EXIT</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>7</number>
      </property>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="inputEN">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);
border-color: rgb(170, 0, 0);
border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="outputEN">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);
border-color: rgb(170, 0, 0);
border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="inputZH">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);
border-color: rgb(170, 0, 0);
border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextBrowser" name="outputZH">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);
border-color: rgb(170, 0, 0);
border-radius: 10px;
line-height: 20px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionAuto_Copy_EN"/>
    <addaction name="actionChatGPT_Stream"/>
    <addaction name="actionCancel_Mouse_Backstage"/>
   </widget>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionProxy"/>
    <addaction name="actionCheck_Proxy"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionFontZH"/>
     <addaction name="actionFontEN"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="actionJust_ZH"/>
    <addaction name="actionJust_EN"/>
    <addaction name="actionRestore"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionManual"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menusettings"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCopyZH">
   <property name="text">
    <string>Copy zhCN</string>
   </property>
  </action>
  <action name="actionAuto_Copy_EN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Copy EN</string>
   </property>
  </action>
  <action name="actionClose_Mouse_Selection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close Mouse Selection</string>
   </property>
  </action>
  <action name="actionChatGPT_Stream">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ChatGPT Stream</string>
   </property>
  </action>
  <action name="actionCancel_Mouse_Backstage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cancel Mouse Backstage</string>
   </property>
  </action>
  <action name="actionZH2EN_only">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zh2en only</string>
   </property>
  </action>
  <action name="actionEN2ZH_only">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>en2zh only</string>
   </property>
  </action>
  <action name="actionBoth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>both</string>
   </property>
  </action>
  <action name="actionProxy">
   <property name="text">
    <string>Proxy</string>
   </property>
  </action>
  <action name="actionJust_ZH">
   <property name="text">
    <string>Just ZH</string>
   </property>
  </action>
  <action name="actionJust_EN">
   <property name="text">
    <string>Just EN</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionCheck_Proxy">
   <property name="text">
    <string>Check Proxy</string>
   </property>
  </action>
  <action name="actionFontZH">
   <property name="text">
    <string>Chinese font</string>
   </property>
  </action>
  <action name="actionFontEN">
   <property name="text">
    <string>English font</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
